scripts/utils/install_detect_secrets.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+[ ! -f "$HOME/.default-python-packages" ] && touch "$HOME/.default-python-packages"
+
+# Array of pip packages to be added
+pip_packages=("virtualenv" "pre-commit" "detect-secrets==1.5.0")
+for package in "${pip_packages[@]}"; do
+  if ! grep -q "$package" "$HOME/.default-python-packages"; then
+    echo "$package" >> "$HOME/.default-python-packages"
+  fi
+done
+
+echo "legacy_version_file = yes" >> "$HOME/.asdfrc"
+asdf plugin add python
+asdf install python latest
+
+if asdf help set &>/dev/null; then
+  # Older asdf version
+  asdf set python latest --home
+else
+  # Newer asdf version
+  asdf global python latest
```

Comment:
I believe its the other way around, `set` is used in the newer version, starting `0.16.0`, while `global` was used in version prior to `0.16.0` release. https://asdf-vm.com/guide/upgrading-to-v0-16.html#asdf-global-and-asdf-local-commands-have-been-replaced-with-asdf-set
```suggestion
if asdf set --help &>/dev/null; then
  # Newer asdf version
  asdf set python latest --home
else
  # Older asdf version
  asdf global python latest
```

Reply:
ahh you are right, thanks